docs/src/pages/customization/palette/palette.md

```diff
@@ -209,6 +209,10 @@ The colors modified by the palette mode are the following:
 
 {{"demo": "pages/customization/palette/DarkTheme.js", "bg": "inline", "hideToolbar": true}}
 
+> Note: The colors are modified only if you use the default palette. If you have a custom palette, you need to make sure that you have the correct values based on the `mode`.
+
+{{"demo": "pages/customization/palette/DarkThemeWithCustomPalette.js", "defaultCodeOpen": false}}
+
```

Comment:
@oliviertassinari @mnajdova Does it make sense to move `dark mode` to its own page?

And I feel like it should be inside `How to guide` more than customization. Or we can also add link in `palette` to the new page.

Reply:
Agree, it could be a guide on it's own

Reply:
- **Standalone page**. I think that it won't make a difference for the built-in search in the docs

<img width="544" alt="Screenshot 2021-09-03 at 10 38 10" src="https://user-images.githubusercontent.com/3165635/131976777-015eab28-ea37-4ae4-a663-f7d661670152.png">

However, if developers are coming from Google, then it makes it easier to find by having a dedicated page title. Material Design has a standalone page: https://material.io/design/color/dark-theme.html, Tailwind has on too.

https://documentation.divio.com/#about-the-system

- **How to Guide vs. Customization**. I would advocate for having it under "Customization" because we have used the "How to Guide" and "Customization" as to [how to](https://documentation.divio.com/#about-the-system) type of content. The main difference is that "How to Guide" is a one-size-fits-all, while "Customization" is on one specific topic.

Reply:
Moved to Customization